File: trunk/MapXtreme/6.6/Samples/VisualStudio2005/Desktop/Features/Find/vb/MapForm1.vb
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms


Namespace FindVB
    '/ <summary>
    '/ Summary description for MapForm1.
    '/ </summary>
    Public Class MapForm1
        Inherits System.Windows.Forms.Form
        Private mapControl1 As MapInfo.Windows.Controls.MapControl
        '/ <summary>
        '/ Required designer variable.
        '/ </summary>
        Private components As System.ComponentModel.Container = Nothing

        Public Sub New(ByVal map As MapInfo.Mapping.Map)
            '
            ' Required for Windows Form Designer support
            '
            InitializeComponent()

            '
            ' TODO: Add any constructor code after InitializeComponent call
            '
            mapControl1.Map = map
        End Sub 'New


        '/ <summary>
        '/ Clean up any resources being used.
        '/ </summary>
        Protected Overloads Overrides Sub Dispose(ByVal disposing As Boolean)
            If disposing Then
                If Not (components Is Nothing) Then
                    components.Dispose()
                End If
            End If
            MyBase.Dispose(disposing)
        End Sub 'Dispose

        '/ Required method for Designer support - do not modify
        '/ the contents of this method with the code editor.
        '/ </summary>
        Private Sub InitializeComponent()
            Me.mapControl1 = New MapInfo.Windows.Controls.MapControl
            Me.SuspendLayout()
            '
            'mapControl1
            '
            Me.mapControl1.Dock = System.Windows.Forms.DockStyle.Fill
            Me.mapControl1.Location = New System.Drawing.Point(0, 0)
            Me.mapControl1.Name = "mapControl1"
            Me.mapControl1.Size = New System.Drawing.Size(292, 266)
            Me.mapControl1.TabIndex = 0
            Me.mapControl1.Text = "mapControl1"
            Me.mapControl1.Tools.LeftButtonTool = Nothing
            Me.mapControl1.Tools.MiddleButtonTool = Nothing
            Me.mapControl1.Tools.RightButtonTool = Nothing
            '
            'MapForm1
            '
            Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
            Me.ClientSize = New System.Drawing.Size(292, 266)
            Me.Controls.Add(Me.mapControl1)
            Me.Name = "MapForm1"
            Me.Text = "MapForm1"
            Me.ResumeLayout(False)

        End Sub 'InitializeComponent

        Private Sub MapForm1_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load '
            mapControl1.Tools.InfoTipsEnabled = True
        End Sub 'MapForm1_Load

        
    End Class 'MapForm1
End Namespace 'FindVB

File: MapXtreme/6.6/Samples/VisualStudio2005/Desktop/Features/Find/cs/Backup/MapForm1.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

namespace FindCS
{
	/// <summary>
	/// Summary description for MapForm1.
	/// </summary>
	public class MapForm1 : System.Windows.Forms.Form
	{
		private MapInfo.Windows.Controls.MapControl mapControl1;
		/// <summary>
		/// Required designer variable.
		/// </summary>
		private System.ComponentModel.Container components = null;

		public MapForm1(MapInfo.Mapping.Map map)
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();

			//
			// TODO: Add any constructor code after InitializeComponent call
			//
			mapControl1.Map = map;
		}

		/// <summary>
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Windows Form Designer generated code
		/// <summary>
		/// Required method for Designer support - do not modify
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
			this.mapControl1 = new MapInfo.Windows.Controls.MapControl();
			this.SuspendLayout();
			// 
			// mapControl1
			// 
			this.mapControl1.Dock = System.Windows.Forms.DockStyle.Fill;
			this.mapControl1.Location = new System.Drawing.Point(0, 0);
			this.mapControl1.Name = "mapControl1";
			this.mapControl1.Size = new System.Drawing.Size(292, 266);
			this.mapControl1.TabIndex = 0;
			this.mapControl1.Text = "mapControl1";
			// 
			// MapForm1
			// 
			this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
			this.ClientSize = new System.Drawing.Size(292, 266);
			this.Controls.Add(this.mapControl1);
			this.Name = "MapForm1";
			this.Text = "MapForm1";
			this.Load += new System.EventHandler(this.MapForm1_Load);
			this.ResumeLayout(false);

		}
		#endregion

		private void MapForm1_Load(object sender, System.EventArgs e)
		{
			mapControl1.Tools.InfoTipsEnabled=true;
		}
	}
}
